speech_to_text.py

```diff
@@ -203,24 +206,6 @@ def finish_job(job: dict) -> dict:
     return job
 
 
-def get_writer_options(job: dict) -> dict:
-    word_options = [
-        "highlight_words",
-        "max_line_count",
-        "max_line_width",
-        "max_words_per_line",
-    ]
-
-    opts = {option: job.get("writer", {}).get(option) for option in word_options}
-
-    # ensure word_timestamps is set if any of the word options are
-    # the timestamps are used by the writer
-    if len(opts) > 0:
-        opts["word_timestamps"] = True
-
-    return opts
-
-
```

Comment:
We might as well pass all writer_options through, so we can set `word_timetamps=True` for the transcription if any are being used. This means this function is no longer needed...